fe/fe-core/src/main/java/com/starrocks/sql/optimizer/cost/CostModel.java

```diff
@@ -301,14 +297,24 @@ public CostEstimate visitPhysicalMergeJoin(PhysicalMergeJoinOperator join, Expre
 
         @Override
         public CostEstimate visitPhysicalNestLoopJoin(PhysicalNestLoopJoinOperator join, ExpressionContext context) {
-            final double nestLoopPunishment = 1000000.0;
             Statistics leftStatistics = context.getChildStatistics(0);
             Statistics rightStatistics = context.getChildStatistics(1);
 
             double leftSize = leftStatistics.getOutputSize(context.getChildOutputColumns(0));
             double rightSize = rightStatistics.getOutputSize(context.getChildOutputColumns(1));
-            return CostEstimate.of(leftSize * rightSize + nestLoopPunishment,
-                    rightSize * StatsConstants.CROSS_JOIN_COST_PENALTY * 2, 0);
+            double cpuCost = leftSize * rightSize + StatsConstants.CROSS_JOIN_COST_PENALTY;
```

Comment:
the old way is multiply CROSS_JOIN_COST_PENALTY， not plus.  if the data size is very larger, plus the CROSS_JOIN_COST_PENALTY seems no effect.

Reply:
HashJoin is `cpuCost = left_carnality + right_cardinality`
NestLoopJoin is `cpuCost = left_cardinality * right_cardinality + COST_PENALTY`

The multiply of cardinality should much larger than the plus ?